less/navigation.less

```diff
@@ -1 +1,41 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+.nav-bar {
+    max-width: 1000px;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
```

Comment:
don't need to specify flex-direction row here since display: flex defaults to row